test/Microsoft.ML.AutoML.Tests/Microsoft.ML.AutoML.Tests.csproj

```diff
@@ -5,6 +5,10 @@
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
```

Comment:
Do we want to create an issue to remember to drop this when we drop net6? Cause that will be happening soon.

Reply:
I already opened https://github.com/dotnet/machinelearning/issues/7268 for tracking.